lib/galaxy/webapps/galaxy/api/uploads.py

```diff
@@ -0,0 +1,42 @@
+"""
+API operations for uploaded files in storage.
+"""
+import logging
+import os
+
+from galaxy import exceptions
+from galaxy.web import expose_api_anonymous
+from galaxy.web.base.controller import BaseAPIController
+
+log = logging.getLogger(__name__)
+
+
+class UploadsAPIController(BaseAPIController):
+
+    @expose_api_anonymous
+    def index(self, trans, **kwd):
+        raise exceptions.NotImplemented("Listing uploads is not implemented.")
+
+    @expose_api_anonymous
+    def create(self, trans, payload, **kwd):
+        """
+        POST /api/uploads/
+        """
+        session_id = payload.get("session_id")
+        session_start = payload.get("session_start")
+        if session_id is None:
```

Comment:
Here we should also check that `session_id` is not empty and doesn't contain unwanted characters (like `.` and `/`), maybe you can reuse this regexp: https://github.com/galaxyproject/galaxy/pull/5516/files#diff-9baf995401cfeb779edf8731ebaf0d2dR526

Reply:
Thx, good idea.